stackslib/src/chainstate/stacks/db/blocks.rs

```diff
@@ -9953,13 +9915,16 @@ pub mod test {
                 let mut debug_reader = LogReader::from_reader(&mut mblock_ptr);
                 let next_mblock = StacksMicroblock::consensus_deserialize(&mut debug_reader)
                     .map_err(|e| {
```

Comment:
Change to

```suggestion
                    .unwrap_or_else(|e| {
```

and remove the `unwrap()` at the end